pkg/config/image_test.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParse(t *testing.T) {
+	// Setup
+	filename := "./testdata/valid_example.yaml"
+	configData, err := os.ReadFile(filename)
+	if err != nil {
+		t.Errorf("Cannot read example YAML file: %v", filename)
+	}
```

Comment:
```suggestion
	require.NoError(t, err)
```

While failing a test with `t.Errorf()` is fine, its execution will still continue unless you also manually include a `return` statement right after. E.g. if we invalidate the example YAML in any way, the test will fail here, however we will also run into nil pointer dereferencing panics on the verifications steps below which will be confusing.

The proposed change will fail and stop the execution, as well as keep up the consistency of using the `testify` module. Also applies to the assertion of the parsing error below.

Reply:
Oh perfect, I was looking for something like this but I didn't find it.